cmd/notary/cert.go

```diff
@@ -63,29 +64,30 @@ func certRemove(cmd *cobra.Command, args []string) {
 	}
 
 	var certsToRemove []*x509.Certificate
+	var certFoundByID *x509.Certificate
 
 	// If there is no GUN, we expect a cert ID
 	if certRemoveGUN == "" {
 		certID := args[0]
 		// This is an invalid ID
-		if len(certID) != idSize {
+		if len(certID) != notary.IDSize {
```

Comment:
Seems inflexible.

Without these size checks, an old client would happily accept newer TUF files that used a longer hash if we wanted to increase the size. With the check, they won't.